docs/site-example/site.conf

```diff
@@ -69,6 +69,8 @@
   -- for channel.
   wifi5 = {
     channel = 44,
+    outdoor_chanlist = '100-140',
+    outdoor_preset = true;
```

Comment:
`;` should be a `,`.

How about making this a tristate (true = default to outdoor mode enabled, false = default to disabled, nil = default is device-dependent)?

Reply:
So rather than `outdoor_preset` this would become `outdoor` with states 
- `true` all devices with 5 GHz use outdoor channels (eliminates 5 GHz mesh completely)
- `false` no device uses outdoor channels (same as not defining `outdoor_chanlist`)
- `nil` preset devices use outdoor channels

Reply:
One alternative option would be to replace the `nil` with a string like `'preset'` (where 'preset' is the default, usually making nil equivalent to 'preset'). This would allow domain configs to override a true/false site config with an explicit 'preset' option.

Reply:
Tristate done.